### ISSUE TITLE
Planning for the 2020 CF meeting: Santander, 9-11 June

### ISSUE THREAD

**Issue by davidhassell** [cf-convention/discuss#35]
Dear CF community,

Further to the save-the-date announcement (#12), the dates for this year's CF meeting in Santander, Spain are now fixed: **10-11 June 2020**

In addition we are planning a half-day training event on the preceding afternoon:  **09 June 2020**

The dates are fixed, however the content is not finalised. The broad areas we expect to cover include

- governance,
- unresolved proposals for enhancements to CF, and
- CF procedures (e.g. GitHub, standard names, ...),

and any specific suggestions on these topics, or additional topics, will be most welcome.

The half-day training event will focus on how to interact with CF systems and processes, such as how to use GitHub to suggest enhancements, and how best to get new standard names accepted. Again, please let us know of specific items that you might find useful for this part of the workshop.

Many thanks, and we hope to see you there (in person or virtually),

David Hassell, on behalf of the organising committee

**Comment by jen-thomas** [comment]
Dear @davidhassell 

Thanks for your organisation of the meeting. I'm new to CF and would really like to attend the training but would only be able to do so virtually. I just wanted to ask if remote access will be possible to the training as well as the meeting itself please? 

Looking forward to hearing more.
Thanks very much!

**Comment by davidhassell** [comment]
Hello @jen-thomas,

There will certainly be remote access for the training afternoon on the 9th June. Any presented information will be broadcast, but we're not sure as yet how that may work for any practical activities - it's still early in the planning process and we'll know more in the coming weeks.

Hope that helps
David

**Comment by jen-thomas** [comment]
Ok that's very helpful, thanks!

**Comment by davidhassell** [comment]
Hello,

Due to the global Covid-19 situation, we have decided to cancel the physical workshop in Santander, 09-11 June. I suspect that this will not come as a surprise, but is clearly the right thing to do.

The good news is that we will still have a workshop this year, but we need some more time to decide on the format (and time), one that suits the typical nature of CF meetings. A virtual conference is clearly a possibility - many of us will already have had some experience with wholly virtual conferences, and from my personal perspective, the one I attended last week worked very well - but it is not necessarily the only option.

Please let us know if you have any suggestions or experiences that may help us decide what to do. In any case, I expect that we will have come up a new plan by the end of April, at the latest. 

Until later, all the best,

David, on behalf of the organising committee

**Comment by davidhassell** [comment]
Hello,

We now, finally, have a plan for the CF meeting. Thank you for your patience.

* The meeting will be virtual.
* The dates are unchanged: **9-11 June 2020**
* The meeting will run for only **3 hours on each day**: **16:00 to 19:00 UTC**, followed by a short purely social time for those who can, and want to, stay on.
* Each day will have a main theme:
  - **Governance**
  - **Unresolved proposals for enhancements to CF**
  - **CF procedures**

As ever, we want to maximise discussion time, so each session will comprise short, live presentations followed by plenty of time for questions, and hopefully to resolve at least some issues.

We are still working on a more detailed agenda, and will post a draft soon. In the mean time, if there are any suggestions on a particular topics for discussion (including offers to present and lead debates) then they are most welcome.

The proposed training event (which was to have run on 9th June) will no longer happen, I'm sorry to say, but the third session on CF procedures will overlap with this activity. Training materials will still be created at a later date, and will surely be informed by the discussion on the 11th June.

All the best,
David, on behalf of the organising committee

**Comment by larsbarring** [comment]
Dear all,

I would like to suggest that the discussion in [cf-conventions #197](https://github.com/cf-convention/cf-conventions/issues/197) would be brought onto the agenda as an unresolved proposal. As far as I recall, without reading up on details, there were a couple of alternative proposals put on the table:

- a minimal change (at least in some sense) to just deprecate the attribute `climatology` to keep present de facto usage of cell methods `within month` and `over years` (etc) in CMIP6 in line with CF, 
- a suggestion to decouple these specific cell methods from the `climatology` attribute
- a suggestion to introduce an entirely new mechanism allowing for a more flexible description of multistep temporal processing of the data

I  think that this would be timely as CMIP7 specs is currently in the works, and this was brought up as [somthing to deal with](https://github.com/cmip6dr/cmip7_forward_look/issues/48).

Kind regards,
Lars

**Comment by davidhassell** [comment]
Dear CF community,

We have time during the CF meeting in June to discuss a number of ongoing suggestions for enhancements with the aim of moving the discussions on. This might be useful to increase the awareness of an enhancement that would benefit from a wider audience, or where the debate has perhaps stalled with text-only communication.

We would like to invite suggestions for which ongoing issues to include.

The format during the meeting would be a short (10 minutes) summary of the enhancement and what the outstanding issues are, followed by a longer (30-45 minutes) open discussion to try to resolve any questions, or at least progress the discussion so that it can carry on more fruitfully after the meeting. Unfortunately the meeting is quite short, so we will only have time to discuss at most four or five issues in this way. 

If you would like to volunteer to "champion" an issue at the meeting, by presenting it and leading the subsequent discussion as described above, then please let us know by replying to this message.

If there are more offers than time allows then we will collectively decide on which ones are best covered in the meeting, not based on the order in which the offers came.

Below is a list of all of the currently open issues from which to draw in which we have included the initial proposer and assignees, not to limit the invitation to them, but simply to increase the visibility of this request.

Many thanks,
David, on behalf of the organising committee

# The issues

Standard way to define subsampled coordinates https://github.com/cf-convention/discuss/issues/#37 @oceandatalab

Metadata handling through processes https://github.com/cf-convention/discuss/issues/#33 @huard

"mesh variable" instead of "boundary variable" for contiguous grid cells https://github.com/cf-convention/discuss/issues/#5 @rabernat

Clarification of alternative coordinates https://github.com/cf-convention/discuss/issues/#4 @taylor13

Usage of `positive` attribute for vertical alternative coordinates https://github.com/cf-convention/discuss/issues/#3 @neumannd

Clarification of requirements on calendar attribute of a bounds variable enhancement
https://github.com/cf-convention/cf-conventions/issues/265 @martinjuckes

udunits supports ppm, but documentation states it does not enhancement https://github.com/cf-convention/cf-conventions/issues/260 @mathiasbockwoldt

Update geostationary projection to allow clean description of newer generation satellites enhancement https://github.com/cf-convention/cf-conventions/issues/258 @erget @JonathanGregory

Procedural items GitHubProblem https://github.com/cf-convention/cf-conventions/issues/257 @erget
@JonathanGregory

Clarification on x/y-grid direction in standard names question https://github.com/cf-convention/cf-conventions/issues/252 @hrajagers

Proposal: Example for Independent Latitude, Longitude, Non-spatiotemporal Variable, and Time Axes enhancement https://github.com/cf-convention/cf-conventions/issues/249 @GeyerB

Add direction attribute for coordinate axis enhancement https://github.com/cf-convention/cf-conventions/issues/247 @snowman2

Geostationary projection & latitude_of_projection_origin question https://github.com/cf-convention/cf-conventions/issues/246 @snowman2

Add earth shape parameters to geostationary projection enhancement https://github.com/cf-convention/cf-conventions/issues/241 @erget @JimBiardCics

Remove restrictions on netCDF object names enhancement https://github.com/cf-convention/cf-conventions/issues/237
@Dave-Allured

Allow CRS WKT to represent the CRS without requiring comparison with grid mapping parameters enhancement https://github.com/cf-convention/cf-conventions/issues/222 @snowman2

CF-checker warnings when following example H.11 (Atmospheric sounding profiles) question
https://github.com/cf-convention/cf-conventions/issues/211 @GeyerB @RosalynHatcher

Typos in Examples H.9 to H.11 -Atmospheric sounding profiles typo https://github.com/cf-convention/cf-conventions/issues/210 @GeyerB

Proposal: Add optional `flag_methods` variable attribute enhancement https://github.com/cf-convention/cf-conventions/issues/205 @jessicaaustin

A convention for complex numbers? question https://github.com/cf-convention/cf-conventions/issues/204 @shoyer

Cell methods: "within"|"over" "days"|"months" and time axis (Section 7.4) question
https://github.com/cf-convention/cf-conventions/issues/197 @larsbarring

Question on Features and feature types question https://github.com/cf-convention/cf-conventions/issues/196 @ngalbraith

Adding figure to paragraph "Bounds for 2-D coordinate variables with 4-sided cells" in Section 7.1 on bounds enhancement https://github.com/cf-convention/cf-conventions/issues/193 @neumannd

State a position on new NetCDF features not explicitly covered by the CF Conventions enhancement https://github.com/cf-convention/cf-conventions/issues/191 @MTG-Formats

Vertical coordinates when only the bounds of the cells are of interest enhancement https://github.com/cf-convention/cf-conventions/issues/190 @MaartenSneepKNMI

how to proceed on some issues at cfconventions.org question https://github.com/cf-convention/cf-conventions/issues/188 @graybeal

Updating definition of coordinate variable to account for NUG changes enhancement https://github.com/cf-convention/cf-conventions/issues/174 @martinjuckes

How to Represent "Raw" Sensor output. question https://github.com/cf-convention/cf-conventions/issues/168 @DocOtak

Clarification of time coordinate requirements enhancement https://github.com/cf-convention/cf-conventions/issues/166 @martinjuckes

Formatting errors in table of contents for examples. enhancement https://github.com/cf-convention/cf-conventions/issues/164 @martinjuckes @dblodgett-usgs

grid cells with a varying number of cell bounds https://github.com/cf-convention/cf-conventions/issues/163 @taylor13

forbidden use of missing_value in 2 examples https://github.com/cf-convention/cf-conventions/issues/162 @taylor13

Add attribute citation_id enhancement https://github.com/cf-convention/cf-conventions/issues/160 @castelao @kevin-obrien @jhausman

Incorporating the CF data model into the conventions enhancement https://github.com/cf-convention/cf-conventions/issues/159 @davidhassell @JonathanGregory

Reference UGRID conventions in CF enhancement https://github.com/cf-convention/cf-conventions/issues/153 @rsignell-usgs

Moderation of proposals? https://github.com/cf-convention/cf-conventions/issues/151 @ChrisBarker-NOAA

Add calendars gregorian_tai and gregorian_utc enhancement https://github.com/cf-convention/cf-conventions/issues/148 @JimBiardCics

Add support for attributes of type string enhancement https://github.com/cf-convention/cf-conventions/issues/141 @JimBiardCics @marqh

TRAC https://github.com/cf-convention/cf-conventions/issues/155: Invalid "id" values in CF Standard Name aliasses https://github.com/cf-convention/cf-conventions/issues/132 @mattben

timeSeries featureType with a forecast / reference time dimension? https://github.com/cf-convention/cf-conventions/issues/129 @dblodgett-usgs

DOIs for CF Convention releases? https://github.com/cf-convention/cf-conventions/issues/127
@rsignell-usgs

**Comment by erget** [comment]
For my 0.02€ I advocate discussing "Standard way to define subsampled coordinates" (#37). This was originally raised by @oceandatalab and has moved quite a bit in the past few weeks and I could imagine the preliminary results are very interesting for the wider community.

**Comment by larsbarring** [comment]
And here are my 0.02kr (2 öre): Following on from my [previous post](https://github.com/cf-convention/discuss/issues/35#issuecomment-625378121), if no one else volunteer to introduce [Cell methods: "within"|"over" "days"|"months" and time axis (Section 7.4)](cf-convention/cf-conventions#197). While the discussion has not moved ahead since last autumn, it touches on a key aspects (cell methods) of CF and how to interpret them correctly and use them efficiently.

**Comment by rabernat** [comment]
I still feel that #5 is important. People seem supportive in general, but the issue discussion went down a bit of a technical rabbit hole. Perhaps some f2f [virtual] discussion would help unblock things. I'd be happy to "champion" this issue at the June meeting.

Is registration required for the meeting?

**Comment by huard** [comment]
I suggest folding "Metadata handling through processes [#33](https://github.com/cf-convention/discuss/issues/#33)" into the more general topic of "Tracking provenance information". I can think of a few people who could lead that discussion.

**Comment by snowman2** [comment]
I would be willing to "champion" this issue: "Allow CRS WKT to represent the CRS without requiring comparison with grid mapping parameters enhancement" cf-convention/cf-conventions#222.

**Comment by ChrisBarker-NOAA** [comment]
Im not sure it
s quite the same, but I think it would be good to have a discussion of how these proposals get worked through: cf-convention/cf-conventions#151

It's been some time since I wrote that -- it might be good to step back and see if we need to update the approach to moderating proposals.

**Comment by neumannd** [comment]
Regarding "my" issues:

* [Usage of positive attribute for vertical alternative coordinates](https://github.com/cf-convention/discuss/issues/3):  drop it as topic for the annual meeting; might be more reasonable to create a proper proposal from this issue (when I find time for it)
* [Adding figure to paragraph "Bounds for 2-D coordinate variables with 4-sided cells" in Section 7.1 on bounds enhancement](cf-convention/cf-conventions#193): would be nice, if we could finalize it; does not take much time (collect issues with respect the graphics; accept them; or decide against figures in the conventions document)

**Comment by martinjuckes** [comment]
Hi @davidhassell , @larsbarring , 

I'd like to contribute to the presentation of [Cell methods: "within"|"over" "days"|"months" and time axis (Section 7.4) #197](https://github.com/cf-convention/cf-conventions/issues/197) .. based on my last post to that discussion.

**Comment by davidhassell** [comment]
Thank you very much for all of the suggestions for topics for discussion

The meeting is just over three weeks away, so please post and any extra suggestions that you may have by the end of Wednesday 20th May, to give us time to see how the exact schedule might work.

All the best,
David

**Comment by davidhassell** [comment]
Dear @erget, @larsbarring, @rabernat, @huard, @snowman2, @ChrisBarker-NOAA, @neumannd, @martinjuckes,

Could you please get in touch with me at david.hassell a t ncas.ac.uk so we can discuss what preparations are needed for discussing these topics at the meeting in a couple of weeks. In the first instance, we'll need a short description of the state of each issue that can be read by people in advance of the meeting.

Thank you and all the best,
David

**Comment by davidhassell** [comment]
A new discussion (#53) has been started for further details of the meeting, such as registration and the agenda, now that the format and content have been settled on. Thank you very much for all of your input, and I hope to see you soon,

David, on behalf of the organising committee
